scripts/setup_nfs_ha.sh

```diff
@@ -27,6 +27,13 @@ function setup_required_packages
     systemctl stop nfs-kernel-server
     systemctl disable nfs-kernel-server
 
+    # Setup static port assignments for mountd, nlm (tcp), and nlm (udp) respectively:
+	sed -i 's/^\(RPCMOUNTDOPTS="--manage-gids\)"/\1 -p 2000"/g' /etc/default/nfs-kernel-server
+	cat <<EOF > /etc/sysctl.d/30-nfs-ports.conf
+fs.nfs.nlm_tcpport = 2001
+fs.nfs.nlm_udpport = 2002
+EOF
```

Comment:
I confirmed that creating `/etc/sysctl.d/xxx.conf` for NFS doesn't work, because the `/proc/fs/nfs` directory usually doesn't exist when the `sysctl --system` is run. And I confirmed that creating `/etc/modprobe.d/lockd.conf` with the static ports assigned does work. So I'll revise this code after accepting/merging your PR. Thanks much!